article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -274,13 +219,11 @@ object ArticlePicker {
 
     // include features that we wish to log but not whitelist against
     val features = primaryChecks.mapValues(_.toString) +
-      ("dcrTestGroup" -> testGroup(DotcomRendering)) +
-      ("dcrDiscussionTestGroup" -> testGroup(DiscussionRendering)) +
-      ("userIsInCohort" -> userInMainTest.toString) +
-      ("userIsInCohortDiscussion" -> userInDiscussionTest.toString) +
+      ("dcrTestGroup1" -> testGroup(DotcomRendering)) +
```

Comment:
Can we just leave this as `dcrTestGroup`?

Reply:
Done. (Thank for pointing out)